recipe/meta.yaml

```diff
@@ -22,6 +22,7 @@ requirements:
   run:
     - python >=3.6
     - pytest
+    - py
```

Comment:
Instead of adding `py` could you apply a patch to fix the usage of the deprecated `py.path.local`, see https://github.com/search?q=repo%3Aaragilar%2Fpytest-mpi%20py.path&type=code
The reason for deprecation and how to replace it https://github.com/pytest-dev/pytest/blob/7.4.2/doc/en/deprecations.rst#py-path-local-arguments-for-hooks-replaced-with-pathlib-path.

A patch can rely on the `py.path` vendored by `pytest` https://github.com/pytest-dev/pytest/blob/7.4.2/src/py.py#L7 (an internal implementation https://github.com/pytest-dev/pytest/blob/cac1eed0ea2901c6acc249cdd0050fe61caf1e85/src/_pytest/_py/path.py#L1473)

Reply:
Should we keep it in for now since need to remove the `py` hotfix as soon as possible?

If anything we can come back and patch next build?

Reply:
Yes, keep it